examples/LedStrip/LedStrip.ino

```diff
@@ -1,9 +1,9 @@
 #include <Homie.h>
 
-const unsigned char NUMBER_OF_LED = 4;
+const unsigned int NUMBER_OF_LED = 4;
 const unsigned char LED_PINS[NUMBER_OF_LED] = { 16, 5, 4, 0 };
 
-HomieNode stripNode("strip", "strip");
+HomieNode stripNode("strip", "Strip", "strip", "true", 1, NUMBER_OF_LED);
```

Comment:
The constructor signature is `HomieNode(const char* id, const char* name, const char* type, bool range, uint16_t lower, uint16_t upper, const NodeInputHandler& inputHandler)`
Shouldn't that just be `true` instead of `"true"` for the type?

Reply:
Absolutely correct :+1: